language/compiler/ir-to-bytecode/syntax/src/syntax.rs

```diff
@@ -60,6 +60,31 @@ fn consume_token<'input>(
     Ok(())
 }
 
+fn parse_comma_list<'input, F, R>(
+    tokens: &mut Lexer<'input>,
+    list_end_token: Tok,
+    parse_list_item: F,
+    allow_trailing_comma: bool,
```

Comment:
Mega-nit, I might have done
```rust
    parse_list_item: F,
    allow_trailing_comma: bool,
    list_end_token: Tok,
```
As it matches how the comma list is laid out more:  `<item>, <end>`

Reply:
I suppose. I expect to add some more parameters related to error handling, so maybe I'll shuffle them around at that time.